src/components/donation/Steps/Submit/PaypalCheckout/Checkout.tsx

```diff
@@ -77,6 +69,14 @@ export default function Checkout({ orderId, onBack }: Props) {
               return orderId;
             }}
           />
+          <button
```

Comment:
this now looks
<img width="635" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/fd532150-66b6-4e31-a48a-7b33ad17fab1">